tests/v3/expected/github.ts

```diff
@@ -1672,8 +1672,8 @@ export interface operations {
             pem: string;
           } & { [key: string]: any });
       };
-      404: unknown;
-      422: unknown;
+      404: components["responses"]["not_found"];
+      422: components["responses"]["validation_failed_simple"];
```

Comment:
@gr2m how does this sit with you? Is this better?

Reply:
looks great!

Reply:
looks great!